crates/environ/src/lib.rs

```diff
@@ -87,7 +86,26 @@ macro_rules! for_each_libcall {
             (CeilF64, wasmtime_f64_ceil),
             (FloorF64, wasmtime_f64_floor),
             (TruncF64, wasmtime_f64_trunc),
-            (NearestF64, wasmtime_f64_nearest)
+            (NearestF64, wasmtime_f64_nearest),
+            (Probestack, wasmtime_probestack),
+            (Memcpy, wasmtime_memcpy),
+            (Memset, wasmtime_memset),
+            (Memmove, wasmtime_memmove),
```

Comment:
Out of curiosity, where do these get generated in cranelift?

Reply:
I assume they will be generated when implementing the bulk memory wasm proposal.